quantus/functions/explanation_func.py

```diff
@@ -46,7 +46,7 @@
     import tf_explain
 
 
-def explain(model, inputs, targets, **kwargs) -> np.ndarray:
+def explain(model, inputs, targets, **kwargs) -> Union[np.ndarray, torch.Tensor]:
```

Comment:
This would require users of tensorflow to also have torch, which is not desirable. A user should be able to choose either framework of choice

Reply:
The alternative would be to remove the return typing for this function...

Reply:
This probably can be avoided by 
```
from __future__ import annotations

from typing import TYPE_CHECKING

if TYPE_CHECKING:
    import tensorflow as tf
    import torch
```

In nutshell, because of `from __future__ import annotations` annotations will be evaluated as strings during runtime, 
but as classes by linters/type checkers/IDEs. Moreover, you could use more compact syntax `np.ndarray | torch.Tensor | tf.Tensor`.

Furthermore, I think we could have parameter `return_tensors` similar to [huggingface](https://huggingface.co/transformers/v3.5.1/main_classes/tokenizer.html), which allows users to choose which framework's tensors should be returned.

And lastly, if this type will be used everywhere, why don't we define a `TypeAlias` for it? Something like `Tensor = Union[np.ndarray, torch.Tensor, tf.Tensor]`.

Reply:
@vedal any comments on @aaarrti suggestion?